addon/plugins/article-structure-plugin/utils/structure.ts

```diff
@@ -10,11 +10,19 @@ import {
   ELI,
   RDF,
   SAY,
+  XSD,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/utils/constants';
 import {
   hasRDFaAttribute,
   Resource,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/utils/namespace';
+import { unwrap } from '@lblod/ember-rdfa-editor-lblod-plugins/utils/option';
+import { StructureSpec } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/article-structure-plugin';
+import {
+  romanize,
+  romanToInt,
+} from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/article-structure-plugin/utils/romanize';
+import { DOMOutputSpec } from 'prosemirror-model/src/to_dom';
```

Comment:
The error seems to occur due to this line, which directly imports from a module in `prosemirror-model`. I would change this to an import from the `@lblod/ember-rdfa-editor` package.